### PR TITLE
Revamp form vitamin request

### PR DESCRIPTION
**Pull request by maulanayuseph**
**Changes**
- Update progress header component
- Create a new file `firstStepVitamin.js`
- Add emit `consultation`
- Update params request type

**Preview**
Web capture_10-3-2022_125634_localhost

**Evidence**
title: Revamp form vitamin request
project: Pikobar Website
participants: @maulanayuseph @Ibwedagama @bangunbagustapa @yoslie

### CODE REVIEW

File: components/_pages/index/ConsultationVitamin/ProgressHeader.vue [hunk 1]
```diff
@@ -25,11 +25,21 @@ export default {
...
   computed: {
     img () {
       return require(`~/assets/progressbar/progress-${this.step}.svg`)
+    },
+    title () {
+      return (this.consultation) ? 'Konsultasi Dokter' : 'Pengajuan Vitamin'
+    },
+    subTitle () {
+      return (this.consultation) ? 'Konsultasi dengan dokter mengenai gejala yang Anda rasakan' : 'Ajukan vitamin ketika Anda sedang melakukan isolasi mandiri dengan mengisi form ini'
     }
```
Comment (Ibwedagama) [inline components/_pages/index/ConsultationVitamin/ProgressHeader.vue:43]: minor:

hello @maulanayuseph , from this reference: https://dictionary.cambridge.org/dictionary/english/subtitle , the `subtitle` is actually a valid word so I think we don't have to use camel case

**Review by Ibwedagama — APPROVED** [review]

File: components/_pages/index/ConsultationVitamin/ProgressHeader.vue [hunk 2]
```diff
@@ -25,11 +25,21 @@ export default {
...
+      type: Boolean,
+      default: false
     }
   },
   computed: {
     img () {
       return require(`~/assets/progressbar/progress-${this.step}.svg`)
+    },
+    title () {
+      return (this.consultation) ? 'Konsultasi Dokter' : 'Pengajuan Vitamin'
```
Comment (bangunbagustapa) [inline components/_pages/index/ConsultationVitamin/ProgressHeader.vue:39]: Looks good, just wonder why this expression is in parentheses.

**Review by bangunbagustapa — APPROVED** [review]

File: components/_pages/index/ConsultationVitamin/ProgressHeader.vue (hunk 1 above)
Comment (maulanayuseph) [reply components/_pages/index/ConsultationVitamin/ProgressHeader.vue]: sorry it's my bad thanks @Ibwedagama  fixed on this [commit](https://github.com/jabardigitalservice/pikobar-jabarprov-go-id/pull/480/commits/56596f4fb07433009c440d4e81c2a962bd2d0c75)

File: components/_pages/index/ConsultationVitamin/ProgressHeader.vue (hunk 2 above)
Comment (maulanayuseph) [reply components/_pages/index/ConsultationVitamin/ProgressHeader.vue]: thanks simplified on this [commit](https://github.com/jabardigitalservice/pikobar-jabarprov-go-id/pull/480/commits/56596f4fb07433009c440d4e81c2a962bd2d0c75)

**Review by yoslie — APPROVED** [review]

**Pull request merged**